posydon/interpolation/profile_interpolation.py

```diff
@@ -134,9 +137,14 @@ def scrape(self,grid,ind,hms_s2):
         
         if 'omega' in self.names:
             profiles[-1]= profiles[self.names.index('omega')]/  \
-                          grid.final_values['S1_surf_avg_omega_div_omega_crit'][ind]
+                          (grid.final_values['S1_surf_avg_omega'][ind]/ \
+                           grid.final_values['S1_surf_avg_omega_div_omega_crit'][ind])
         
-        return scalars, profiles
+        if not np.isnan(grid.final_values['S1_surf_avg_omega_div_omega_crit'][ind]):
+            return scalars, profiles
+        else:
+            Pwarn("nan in final values, binary will not be"
+                  "included in file","InappropriateValueWarning")
```

Comment:
I'm not sure, but maybe an "IncompletenessWarning" is better here, because you like to point out, that stuff won't be added and not that there are `nan` values in the data. This is a borderline case, where both warning types apply and you should chose the one you think is more appropriate to transport your main aim.

Reply:
good point. I'll change to IncompletenessWarning.